src/Common/IntegrationTestCategory.cs

```diff
@@ -0,0 +1,13 @@
+// This file was auto-generated from 'constants.yaml'. Changes may be overridden.
+
+namespace Microsoft.Oryx.Common
+{
+    public static class IntegrationTestCategory
```

Comment:
I thought we could enhance the SharedCodeGenerator so that it actually outputs an `enum`, not a class with `const string`s :)